### PR TITLE
Wheelchair fix

### LINKED ISSUE #14754: Motorized Wheelchair cell usage is bugged
(WEB REPORT BY: dragonbro REMOTE: 172.93.109.202:7777)
>Revision (Should be above if you're viewing this from ingame!)
cc5cd8c387ca0fae9f66061196d8627ac9bbca33 

>General description of the issue
The way the motorized wheelchair draws power from the cell is flawed, leading it to run endlessly

>What you expected to happen
Should the wheelchair's cell charge be below the consumption rate, it would stop working

>What actually happened
It can run along nicely and infinitely on 1 charge (Movement consumption is 2), thus leading to unlimited speedy wheelchair usage

>Steps to reproduce if possible
Literally just ride around in the wheelchair, watching the cell charge usage

### PR DESCRIPTION
**Pull request by Kurfursten**
fixes #14754

wheelchairs can move indefinitely if charge becomes negative

### CODE REVIEW

**Comment by ghost** [comment]
That doesn't quite fix the issue.

I found that if you set the cell charge to something like 5, it won't tick to 0. I think cell.use is supposed to return a 0 if it's not capable of doing so (because there's not enough charge), might be best to use that?

**Comment by Kurfursten** [comment]
I'll look into it in detail later then, kinda busy today going to an anniversary

File: code/game/objects/structures/vehicles/wheelchair.dm [hunk 1]
```diff
@@ -220,13 +220,13 @@
 		internal_battery.use(2) //Example use: 100 charge to get from the cargo desk to medbay side entrance
```
Comment (Intigracy) [inline code/game/objects/structures/vehicles/wheelchair.dm:220]: it should be
```
if(internal_battery && internal_battery.use(2)
   ..()
```

**Comment by ghost** [comment]
>wheelchair fix [needs fixing]
:thinking:

File: code/game/objects/structures/vehicles/wheelchair.dm (hunk 1 above)
Comment (Kurfursten) [reply code/game/objects/structures/vehicles/wheelchair.dm:220]: That's not quite right. The wheelchair doesn't lock up entirely if out of batteries, you can still move it by hand (but it's even slower than a normal wheelchair).

**Comment by Kurfursten** [comment]
I'm gonna go ahead and close this since I am not actively coding at the moment

**Pull request closed without merging**